presto-jdbc/src/test/java/com/facebook/presto/jdbc/TestJdbcPreparedStatement.java

```diff
@@ -101,22 +100,6 @@ public void testExecuteQuery()
         }
     }
 
-    @Test
```

Comment:
Please keep this as a JDBC unit test. We should only use product tests for additional things that cannot be tested as normal unit test, as they are much harder to run, take longer, and are far away from the actual code being tested. Ideally, building `presto-jdbc` will test as much as possible for the driver.

Reply:
The idea was to cover scenario also for simba drivers, maybe would have both tests.

Reply:
That seems reasonable.

Reply:
Okay, will make 2 tests, and will adorn the product test with some explanation